entity-service/src/main/kotlin/com/egm/stellio/entity/config/WebConfig.kt

```diff
@@ -1,15 +1,26 @@
 package com.egm.stellio.entity.config
 
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
 import org.springframework.http.codec.ServerCodecConfigurer
+import org.springframework.web.reactive.config.CorsRegistry
 import org.springframework.web.reactive.config.EnableWebFlux
 import org.springframework.web.reactive.config.WebFluxConfigurer
 
+@Profile("!test")
```

Comment:
why this exclusion?

Reply:
Because when I added allowed.origin it breaks some unit tests. `Could not resolve placeholder 'allowed.origins' in value "${allowed.origins}"`

Reply:
I can remove the exclusion and add allowed.origins to application-test.properties

Reply:
or add a default in application.properties

Reply:
done